app/screens/home/EditChildProfile.tsx

```diff
@@ -353,6 +353,7 @@ const EditChildProfile = ({ route, navigation }: Props): any => {
     const insertData: any = editScreen
       ? await getNewChild(
         uuid,
+        "false",

```

Comment:
We should use a variable instead of magic stings.
For example: isDefaultChild

Reply:
okay